packages/slate-editor/src/modules/editor/plugins/withElementsEqualityCheck.ts

```diff
@@ -15,20 +29,100 @@ export interface ElementsEqualityCheckEditor {
     isElementEqual(node: Element, another: Element): boolean | undefined;
 }
 
-export function withElementsEqualityCheck(getExtensions: () => Extension[]) {
-    return function <T extends SlateEditor>(editor: T): T & ElementsEqualityCheckEditor {
-        function isElementEqual(node: Element, another: Element): boolean | undefined {
-            for (const extension of getExtensions()) {
-                const ret = extension.isElementEqual?.(node, another);
-                if (typeof ret !== 'undefined') {
-                    return ret;
-                }
-            }
-            return undefined;
+export function withElementsEqualityCheck<T extends SlateEditor>(
+    editor: T,
+): T & ElementsEqualityCheckEditor {
+    return Object.assign(editor, {
+        isElementEqual,
+    });
```

Comment:
Potentially the function could have just become a lib function (and not a method of `editor`). But I wanted to make the minimal necessary changes at this stage.